qutebrowser/utils/usertypes.py

```diff
@@ -113,18 +114,39 @@ def _snap_in(self, offset):
             True if the value snapped in (changed),
             False when the value already was in the list.
         """
-        op = operator.le if offset < 0 else operator.ge
+        optype = operator.le if offset < 0 else operator.ge
+        currentval = None
+        items = None
+
+        is_int = lambda s: isinstance(self.items[self._idx], int) \
+                or s.lstrip("-+").isdigit()
```

Comment:
Line 121 checks integers, line 122 checks for strings. test_cmdhistory.py has some cases that use strings in the neighborlist, like test_nextitem_previtem_chain I believe. It is like: ['first', 'second', 'third', 'fourth', 'fifth']. In that case it worked fine for it to instead go into line 138 and have it sort the list; it can't go into line 136 and do some subtraction math.